sass/themes/donate/components/header/_header.scss

```diff
@@ -1,8 +1,8 @@
 header[role=banner] {
 
   &.header {
-    height: 50px;
-    padding: 4px 0;
+    min-height: 50px;
+    padding: 9px 0;
     border-width: 2px;
```

Comment:
I quickly checked what it would look like and noticed that somewhere else the padding-bottom gets overridden to 0 with an important flag. So the 9px bottom padding is not coming through.